datastore_entity_derives/src/lib.rs

```diff
@@ -182,11 +196,10 @@ pub fn datastore_managed(input: TokenStream) -> TokenStream {
                 return Ok(result)
             }
             #(
-                // TODO - only do this for indexed properties
                 pub fn #entity_getters<A>(value: #entity_getter_key_types, token: A, project_name: &String) -> Result<#name, String> 
                     where A: ::google_api_auth::GetAccessToken + 'static
                 {
-                    let datastoreEntity = datastore_entity::get_one_by_property(#keys.to_string(), value, #kind_str.to_string(), token, project_name)?;
+                    let datastoreEntity = datastore_entity::get_one_by_property(#ds_property_names.to_string(), value, #kind_str.to_string(), token, project_name)?;
```

Comment:
`datastore_entity`

Reply:
ah! yes yes! snake snake snake case

Reply:
fixed in https://github.com/andreaspsson/datastorers/pull/16